Sming/Windows.mk

```diff
@@ -0,0 +1,34 @@
+#	Windows.mk
+#
+#==============================================================================
+# 	default directories (make compatible format /<drive>/<absolute path>)
+#------------------------------------------------------------------------------
+# 	ESP8266 SDK directories
+ESP_HOME 		?= c:/Espressif
+ESP_HOME 		:= $(subst \,/,$(addprefix /,$(subst :,,$(ESP_HOME))))
+
+SDK_BASE        := $(ESP_HOME)/ESP8266_SDK
+SDK_TOOLS       := $(ESP_HOME)/utils/ESP8266
+
+#==============================================================================
+# 	default serial settings
+#------------------------------------------------------------------------------
+COM_PORT		?= COM3
+
+#==============================================================================
+# 	tool settings
+#------------------------------------------------------------------------------
+ESPTOOL		 	?= $(SDK_TOOLS)/esptool.exe
+GET_FILESIZE 	?= stat --printf="%s"
+MEMANALYZER  	?= $(SDK_TOOLS)/memanalyzer.exe $(OBJDUMP).exe
```

Comment:
This will make python a new required for windows builds. Is there any advantage of the python version?